pkg/volume/util/util.go

```diff
@@ -192,3 +193,38 @@ func CheckNodeAffinity(pv *v1.PersistentVolume, nodeLabels map[string]string) er
 	}
 	return nil
 }
+
+// ValidateLocalPath validates a single path, returning an error if the path is
+// invalid. Paths may not:
+//
+// 1. be relative
+// 2. contain '..' as an element
+// 3. start with '..'
+// 4. contain filename larger than 255 characters
+// 5. be longer than 4096 characters
+func ValidateLocalPath(targetPath string) error {
```

Comment:
Why is this check different from 

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/validation.go#L947

It seems weird to me to have different validation here than every where else.